lib/src/component2_suggestors/component2_constants.dart

```diff
@@ -35,3 +35,28 @@ String getDeperecationMessage(String methodName) {
   ///
   $revertInstructions''';
 }
+
+final componentWillMountMessage = '''
```

Comment:
#nit this could be `const` instead:
```suggestion
const componentWillMountMessage = '''
```